concepts/05 UI Components/HtmlEditor/00 Getting Started with HtmlEditor/20 Customize the Toolbar.md

```diff
@@ -0,0 +1,230 @@
+The HtmlEditor's toolbar contains buttons and drop-down menus that allow users to edit and format content. To customize the toolbar, specify an array of [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) in the [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/) object. You can use [predefined items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Predefined_Items/) or create [custom items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Add_a_Custom_Item/). Specific predefined items, such as font size, support multiple values. In these cases, use the [name](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#name) property to define the item and the [acceptedValues](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#acceptedValues) property to assign an array of values that should be available.
```

Comment:
> Shouldn't it be 'selectboxes' instead of drop-down menus?

Select boxes and drop-down menus are the same thing, but I think the term "select box" is less common.

> Maybe replace 'customize' with 'configure'?

> assign an array of available values?

OK

```suggestion
The HtmlEditor's toolbar contains buttons and drop-down menus that allow users to edit and format content. To configure the toolbar, specify an array of [items](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) in the [toolbar](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/) object. You can use [predefined items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Predefined_Items/) or create [custom items](/Documentation/Guide/UI_Components/HtmlEditor/Toolbar/Add_a_Custom_Item/). Specific predefined items, such as font size, support multiple values. In these cases, use the [name](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#name) property to define the item and the [acceptedValues](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/#acceptedValues) property to assign an array of available values.
```